client/src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -46,9 +46,12 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
     >
       {({ handleSubmit, handleChange, values, touched, errors, isSubmitting }) => (
         <form onSubmit={handleSubmit} className={classes.form} noValidate>
+          {/* moved lables out of input fields to resemble mockup */}
+          <Typography fontSize={10} fontWeight={'bolder'} className={classes.label}>
+            EMAIL ADDRESS
```

Comment:
The text transform should be used for uppercase. This is for accessibility, screen readers read these out as abbreviations.